assets/tests/components/alertsPage.test.tsx

```diff
@@ -86,4 +136,27 @@ describe("Alerts Page", () => {
       });
     });
   });
+
+  describe("Alert Places List", () => {
+    test("navigating to / from the places list for an alert", async () => {
+      const { getByText, getByTestId, queryByText } = render(
+        <AlertsPage
+          places={placesAndScreens as Place[]}
+          screensByAlertId={alertsOnScreens as ScreensByAlert}
+          isVisible={true}
+        />
+      );
+
+      await act(async () => {
```

Comment:
Not sure what the problem is, but I'm having trouble getting this test to pass. For some reason, the test doesn't think any alerts are in the document when it tries to click `10`.